packages/features/ee/common/components/v2/LicenseRequired.tsx

```diff
@@ -27,20 +29,20 @@ const LicenseRequired = ({ children, as = "", ...rest }: LicenseRequiredProps) =
       ) : (
         <EmptyScreen
           Icon={Icon.FiAlertTriangle}
-          headline="This is an enterprise feature"
+          headline={t("enterprise_license")}
           description={
-            <>
-              To enable this feature, get a deployment key at{" "}
-              <a href={CONSOLE_URL} target="_blank" rel="noopener noreferrer" className="underline">
+            <div
+              dangerouslySetInnerHTML={{
```

Comment:
Can we not do it without using `dangerouslySetInnerHTML` ?

Reply:
another approach would be with vanilla js. I added a sanitize funciton to the html string so it should be fine